databases/week3/hyf_homework_databases_lesson3.sql

```diff
@@ -0,0 +1,166 @@
+-- Data model
+CREATE DATABASE hyf_meal_sharing;
+USE hyf_meal_sharing;
+
+CREATE TABLE `meal` (
+`id` int unsigned NOT NULL PRIMARY KEY AUTO_INCREMENT,
+`title` varchar(255) NOT NULL,
+`description` text NOT NULL,
+`location` varchar(255) NOT NULL,
+`when` datetime NOT NULL,
+`max_reservations` int unsigned NOT NULL,
+`price` decimal NOT NULL,
```

Comment:
It's a better practice to specify the number of significant digits that are stored and the number of digits that can be stored following the decimal point: https://dev.mysql.com/doc/refman/8.0/en/fixed-point-types.html

Reply:
Good to know thanks a lot!